app/views/shared/vacancy/_jobseeker_view.html.haml

```diff
@@ -5,7 +5,10 @@
 
 .jobs-banner
   .govuk-width-container
-    = render 'vacancies/back_link'
+    = render(Shared::BreadcrumbComponent.new(collapse_on_mobile: false,
+                                             crumbs: [{ link_path: root_path, link_text: t('breadcrumbs.home') },
+                                                      { link_path: request.host == URI(request.referrer || '').host ? request.referrer : jobs_path, link_text: t('breadcrumbs.jobs') },
```

Comment:
I'm not sure what `request.host == URI(request.referrer || '').host` is checking. (Is it 'Was the request from this page'?) Can it be wrapped in a boolean method with an explanatory name that lives in the view component?

```
def explanatory_name?(request)
  true
end
```